pkg/artifactory/datasource/repository/local/datasource_artifactory_local_nuget_repository.go

```diff
@@ -0,0 +1,26 @@
+package local
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jfrog/terraform-provider-artifactory/v6/pkg/artifactory/resource/repository/local"
+	"github.com/jfrog/terraform-provider-shared/packer"
+)
+
+func DataSourceArtifactoryLocalNugetRepository() *schema.Resource {
+	constructor := func() (interface{}, error) {
+		return &local.NugetLocalRepositoryParams{
+			RepositoryBaseParams: local.RepositoryBaseParams{
+				PackageType: "nuget",
+				Rclass:      "local",
```

Comment:
Let's use `rclass` here that is defined already in `local.go`

Reply:
I assume we're merging it as is, without this change?

Reply:
Yea, I can update this in my PR.

Reply:
Okay sounds good, i'll merge